packages/@icon-magic/distribute/README.md

```diff
@@ -0,0 +1,104 @@
+# @icon-magic/distribute
+
+This package is responsible for generating the flavors of the icon in all the
```

Comment:
Update the description to what distribute is responsible for. Also need to update the CLI documentation for the distribute command